exonum-java-binding/core/pom.xml

```diff
@@ -342,6 +374,35 @@
               <goal>build-classpath</goal>
             </goals>
           </execution>
+          <execution>
+            <id>copy-core-jar</id>
+            <phase>install</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.basedir}/rust/target/debug/lib/java</outputDirectory>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>com.exonum.binding</groupId>
```

Comment:
Here and elsewhere, use properties please: `${project.groupId}`, etc.

Also, why is this needed, if the code in 'packaging' does a similar thing?